python/tvm/contrib/download.py

```diff
@@ -62,72 +62,74 @@ def download(url, path, overwrite=False, size_compare=False, verbose=1, retries=
                 res_get = urllib2.urlopen(url)
             url_file_size = int(res_get.headers["Content-Length"])
             if url_file_size != file_size:
-                print("exist file got corrupted, downloading %s file freshly..." % path)
-                download(url, path, True, False)
+                if verbose:
+                    print(f"Existing file {path} has incorrect size, downloading fresh copy")
```

Comment:
Is the use of `print()` intentional? as opposed to using the use of `logging` module, similar to other parts of our code base?

One advantage of `logging` is that it can set various log levels, removing the need for the blocks like `if verbose: print("message")`, and instead, we could use `verbose` to set log level into e.g. `DEBUG` in this module.

What do you think?

Reply:
No particular reason for me, other than it being the way the file was previously written.  Looking through the history, there aren't any commits that suggest a need for `print` statements, so I agree that `logging` is the better route to go.

With `logging`, I think that we can drop the `verbose` flag altogether.  The user can still specify whether they want the verbose output by setting that logger's log level, but it doesn't need to be part of the main function call.  For the information in these print statuses, I think they should be at `INFO` level, except for the download progress bar which can be at `DEBUG` level.

Reply:
Yeah, I agree with the log levels you proposed. Thanks!

Reply:
No problem!  I ended up using `WARNING` level a couple of times as well, for the corrupted file and error-retry paths.